src/App/Resources/AppResources.resx

```diff
@@ -1588,6 +1588,24 @@
     <value>Use Biometrics To Unlock</value>
   </data>
   <data name="AccessibilityOverlayPermissionAlert" xml:space="preserve">
-    <value>Bitwarden requires additional accessibility configuration. Open the app for more info.</value>
+    <value>Bitwarden needs attention - See "Auto-fill Accessibility Service" from Bitwarden Settings</value>
+  </data>
+  <data name="BitwardenAutofillServiceOverlayPermission" xml:space="preserve">
```

Comment:
What's the difference in this one vs `BitwardenAutofillServiceStep3`? Seem like almost the same thing.

Reply:
This one is for the toast and directs the user to the app settings for more detail (only 3.5 seconds to get a point across).  Step3 is more verbose since it's displayed on the settings screen.